app/assets/stylesheets/app-vendor.css.less

```diff
@@ -1,5 +1,5 @@
 
-@import "twitter/bootstrap/bootstrap";
+//@import "twitter/bootstrap/bootstrap";
```

Comment:
Let's delete stuff instead of comment it out. If we ever care what used to be there we can look at our history.